.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -49,7 +49,7 @@ runs:
     run: AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_LOCATION=${{ inputs.location }} make az-create-workload-msi
   - name: create cluster
     shell: bash
-    run: AZURE_CLUSTER_NAME=${{ inputs.cluster_name }} AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_ACR_NAME=${{ inputs.acr_name }} AZURE_LOCATION=${{ inputs.location }} make az-mkaks-cilium
+    run: AZURE_CLUSTER_NAME=${{ inputs.cluster_name }} AZURE_RESOURCE_GROUP=${{ inputs.resource_group }} AZURE_ACR_NAME=${{ inputs.acr_name }} AZURE_LOCATION=${{ inputs.location }} make az-mkaks-cniv1
```

Comment:
Can we switch to passing vars in via `env` block? Would improve readability ...

Reply:
Seems out of the scope of this PR but if we want to invest in parameterizing this and maybe some sort of matrix/suite for running the two modes + making these vars in this file come from env instead of passing in we can later.

Reply:
Pushed the changes here, I will try running it and see what happens

Reply:
Worked (https://github.com/Azure/karpenter-provider-azure/actions/runs/11111453449), resolving